bokeh/models/sources.py

```diff
@@ -210,6 +210,18 @@ class RemoteSource(DataSource):
 class AjaxDataSource(RemoteSource):
     method = String('POST', help="http method - GET or POST")
 
+    overwrite = Bool(True, help="""
```

Comment:
You can see lots of examples of using enumerations in `bokeh.models` modules. Let me know if you need more specific pointers.